ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -11,7 +11,7 @@ build_root:
 raw_steps:
 - pipeline_image_cache_step:
     commands: |
-      cargo build --release
+      cargo build --release && git diff --exit-code
```

Comment:
I assume you purposely did not use `cargo build --locked --release` in order to go through the whole build process?

Reply:
> I assume you purposely did not use `cargo build --locked --release` in order to go through the whole build process.

I am not all that familiar with Rust, but my impression is that `--locked` would make the build fail early.  If it shows the changes that it recommends making to `Cargo.lock` so I can apply them locally and update a failing PR, then yeah, that would be better.  But if it just says "your lock file is stale" or something that doesn't tell me how to patch my `Cargo.lock`, then it's not very helpful.  I'm sure the `git diff` approach will tell me how to patch my `Cargo.lock`, so that's what I went with here.

Reply:
rerolled with eb0197004 -> 1ae3cfc09 to get closer to @vrutkovs' [suggestion][1].

[1]: https://github.com/openshift/release/pull/6935#discussion_r373604511